Homework_5.ipynb

```diff
@@ -0,0 +1,651 @@
+{
```

Comment:
<p>Good to add that to the markdown. However what you describe here is not markdown syntax but Latex (eg. https://davidhamann.de/2017/06/12/latex-cheat-sheet/)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_PythonCpp/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>